lib/generators/react_on_rails/install_generator.rb

```diff
@@ -61,13 +62,13 @@ def invoke_generators
       # js(.coffee) are not checked by this method, but instead produce warning messages
       # and allow the build to continue
       def installation_prerequisites_met?
-        !(missing_node? || missing_yarn? || ReactOnRails::GitUtils.uncommitted_changes?(GeneratorMessages))
+        !(missing_node? || missing_npm? || ReactOnRails::GitUtils.uncommitted_changes?(GeneratorMessages))
```

Comment:
Let's remove the whole requirement of npm or yarn as ShakaPacker. We'll make sure one of those exists, or it could be even a different package manager.